packages/gestalt/src/Image.js

```diff
@@ -152,50 +153,42 @@ export default class Image extends PureComponent<Props> {
       srcSet,
     } = this.props;
 
-    const isScaledImage = shouldScaleImage(fit);
     const childContent = children ? (
       <Box position="absolute" top left bottom right overflow="hidden">
         {children}
       </Box>
     ) : null;
 
-    return isScaledImage ? (
-      <Box height="100%" position="relative">
-        <div
-          aria-label={role === 'presentation' ? undefined : alt}
-          className={fit === 'contain' || fit === 'cover' ? styles[fit] : null}
-          style={{
-            backgroundColor: color,
-            backgroundImage: `url('${src}')`,
-          }}
-          role={role}
-        />
-        {childContent}
-      </Box>
-    ) : (
+    const isScaledImage = fit === 'cover' || fit === 'contain';
```

Comment:
should we kill line 7?

Reply:
"line 7"? I'm not sure I follow

Reply:
Yeah, line 7
<img width="775" alt="Screen Shot 2022-11-09 at 11 19 05 AM" src="https://user-images.githubusercontent.com/10593890/200883628-6cb708eb-92f8-4505-972e-5b401abe580a.png">

Reply:
got it, good call